service/raft/raft_group0.cc

```diff
@@ -476,6 +476,7 @@ struct group0_members {
                 ret.push_back(addr.value());
             } else {
                 upgrade_log.debug("get_inet_addrs() - skipping peer {}", srv.addr.id);
+                return {};
```

Comment:
Please squash this patch with the one which introduces `group0_members`.

Reply:
Done.